packages/bilby.cython/meta.yaml

```diff
@@ -0,0 +1,22 @@
+package:
+  name: bilby.cython
+  version: 0.5.3
+  top-level:
+    - docs
```

Comment:
Probably we can remove `docs` from top-level. I think it is too general import name for the package.